zebrad/src/components/mempool.rs

```diff
@@ -188,3 +198,19 @@ impl Service<Request> for Mempool {
         }
     }
 }
+
+fn remove_expired_transactions(
```

Comment:
Can you document this method, and the consensus rule it implements?

See:
https://zips.z.cash/zip-0203#specification

Reply:
https://github.com/ZcashFoundation/zebra/pull/2774/commits/532e3067fd47b38c82edca0188272ba84afd8576